Formula/qscintilla2.rb

```diff
@@ -18,89 +17,81 @@ class Qscintilla2 < Formula
     sha256 cellar: :any, mojave:        "20794671c986947c27acddf7d5770559ac1567a445b452a99af2669089493cc5"
   end
 
-  depends_on "pyqt"
+  depends_on "pyqt-builder" => :build
+
+  # TODO: use qt when octave can migrate to qt6
+  depends_on "pyqt@5"
   depends_on "python@3.9"
   depends_on "qt@5"
   depends_on "sip"
 
-  # Fix for rpath in library install name. Taken from
-  # https://github.com/macports/macports-ports/pull/7790
-  # https://www.riverbankcomputing.com/pipermail/qscintilla/2020-March/001444.html
-  patch :DATA
-
   def install
     spec = (ENV.compiler == :clang) ? "macx-clang" : "macx-g++"
     spec << "-arm64" if Hardware::CPU.arm?
     args = %W[-config release -spec #{spec}]
 
-    cd "Qt4Qt5" do
+    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+
+    cd "src" do
       inreplace "qscintilla.pro" do |s|
+        s.gsub! "QMAKE_POST_LINK += install_name_tool -id @rpath/$(TARGET1) $(TARGET)",
+          "QMAKE_POST_LINK += install_name_tool -id #{lib}/$(TARGET1) $(TARGET)"
```

Comment:
Can we just remove this `QMAKE_POST_LINK`? Based on the patch it doesn't seem to do anything but break things on ARM.

Reply:
Building python binding need it, otherwise it will cause a dyld error: image not found.

Reply:
Why do the Python bindings need them now when they weren't needed before?

Reply:
The `configure.py` is no longer provided by `qscintilla2` so we must use `sip` to build it. `sip-install` will try to build a test program and link against `libqscintilla2_qt5.15.dylib` to ensure `qscintilla2` is installed correctly.

If omit this step, the id of  `libqscintilla2_qt5.15.dylib` is `libqscintilla2_qt5.15.dylib` not `/usr/local/opt/qscintilla2/lib/libqscintilla2_qt5.15.dylib`, so the test program cannot find `libqscintilla2_qt5.15.0.1.dylib` and output is `dyld error: image not found`.